### PR TITLE
New project structure and Angular 8

### PR DESCRIPTION
**Pull request by nnixaa**

### CODE REVIEW

File: DEV_DOCS.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+- [Release](#release)
+
+# Release
+
+0. For major version, search for `@breaking-change` to make sure all breaking changes are covered.
+
+To start a new release (publish the framework packages on NPM) you need:
+
+1. create a new release branch called `release/1.0.2`
```
Comment (Tibing) [inline DEV_DOCS.md]: Let's rename branch here to deal with copy\paste
```suggestion
1. create a new release branch called `release/{version}` 
```
or something like that

File: DEV_DOCS.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+2. `npm run build:lib` to make sure the lib is building
+3. MANUALLY update a version in main ./package.json & in ./packages/ng2-smart-table/package.json
+4. `npm run changelog` to update changelog. Make sure the previous version has a git tag and it is pushed into the origin
+5. commit with `release: v.1.6.0` message
```
Comment (Tibing) [inline DEV_DOCS.md]: The same issues as above

File: DEV_DOCS.md (hunk 1 above)
Comment (Tibing) [reply DEV_DOCS.md]: Or at least use the same version across release guide

File: projects/ng2-smart-table/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Ng2SmartTable
```
Comment (Tibing) [inline projects/ng2-smart-table/README.md]: Do we really need that readme.md file?

File: projects/ng2-smart-table/package.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ng2-smart-table",
+  "version": "1.5.0",
```
Comment (yggg) [inline projects/ng2-smart-table/package.json]: Version changed

**Comment by Tibing** [comment]
LGTM

**Pull request merged**